css/main.css

```diff
@@ -27,7 +27,8 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+#desktop-nav,
+#desktop-nav ul {
   display: none;
 }
 
```

Comment:
- [ ] kindly make sure that the mobile menu looks like the Figma design that appears in the left. You can achieve this by adding `height: 100vh` to the section tag and `align: left`  and `font-style: bold` to the styling properties of this tag.

your design 👇

![design](https://user-images.githubusercontent.com/66992901/190670925-8b9aa551-daac-4950-a7c1-9fa634b2697a.PNG)

Figma design👇

![desfigma](https://user-images.githubusercontent.com/66992901/190670974-6fd1757b-d28d-4622-a044-fbb4fe5b44c5.PNG)